src/libsync/creds/oauth.cpp

```diff
@@ -501,11 +501,19 @@ void OAuth::fetchWellKnown()
         _wellKnownFinished = true;
         Q_EMIT fetchWellKnownFinished();
     } else {
-        qCDebug(lcOauth) << "fetching" << wellKnownPathC;
-
         QNetworkRequest req;
+
+        if (!_webFingerOverrideUrl.isEmpty()) {
```

Comment:
I thought we where getting away without teaching oauth stuff only needed in the wizard?